docs/conf.py

```diff
@@ -13,19 +13,22 @@
 
 import os
 import sys
-from configparser import RawConfigParser
 
 import sphinx_rtd_theme
 
 sys.path.insert(0, os.path.abspath('..'))
 sys.path.append(os.path.dirname(__file__))
 os.environ.setdefault("DJANGO_SETTINGS_MODULE", "readthedocs.settings.dev")
 
-from django.utils import timezone
-
-import django
-django.setup()
+try:
+    # Nesting in try to avoid isort inflicting unwanted changes. It tries to
+    # move these imports above the very necessary sys.path manipulation.
```

Comment:
Looks like we can use pramga comments for this as well: https://pycqa.github.io/isort/docs/configuration/action_comments.html

Reply:
Ah nice, `isort: split` was what I was looking for